lib/features/email/domain/extensions/email_header_extension.dart

```diff
@@ -3,5 +3,5 @@ import 'package:jmap_dart_client/jmap/mail/email/email_header.dart';
 import 'package:tmail_ui_user/features/offline_mode/model/email_header_hive_cache.dart';
 
 extension EmailHeaderExtension on EmailHeader {
-  EmailHeaderHiveCache toHiveCache() => EmailHeaderHiveCache(name: name, value: value);
+  EmailHeaderHiveCache toHiveCache() => EmailHeaderHiveCache(name: name ?? "", value: value ?? "");
```

Comment:
what the `fix` here?

Reply:
it we an error because the name should not be null